changelog.txt

```diff
@@ -1,5 +1,37 @@
 *** Changelog ***
 
+2022-01-12 - version 4.10.0
```

Comment:
Wait, actually, this should be 2023, no?

```suggestion
2023-01-12 - version 4.10.0
```

Reply:
Oops! Suggestions commited. https://github.com/Automattic/sensei/pull/6353/commits/2813272541ab392913c83bbec597cd4c44a3801f